curator-recipes/src/test/java/org/apache/curator/framework/recipes/nodes/TestPersistentEphemeralNode.java

```diff
@@ -634,11 +640,6 @@ private CuratorFramework newCurator() throws IOException
         return client;
     }
 
-    public void killSession(CuratorFramework curator) throws Exception
-    {
-        KillSession.kill(curator.getZookeeperClient().getZooKeeper(), curator.getZookeeperClient().getCurrentConnectionString());
```

Comment:
Why not leave the method and have it do the new thing instead of repeating the same snippet in a bunch of places?